report/2.Analysis/ArduinoInEducation.tex

```diff
@@ -0,0 +1,50 @@
+% point of document
+% We want to explain to the reader that (and how) the arduino platform is used in HTX
+% The point is:
+% - What "teknikfag" is
+% - What "el-teknik" is
+% 
+
+\section{Arduino and their usage in danish education}
+During the third year of the HTX\footnote{Higher Technical Examination Programme, is a 3-year vocationally oriented general upper secondary programme which builds on the 10th-11th form of the Folkeskole\cite{htx_wiki}} programme students are required to take a course in a voluntary technical subject, which utilizes a project-based learning practive. 
+There are a multiple of courses, one of these regard electronics, where many schools choose to utilize the Arduino platform\footnote{based on the fact that 4 of the students in this group studied at HTX in different parts of Denmark and all utilized the Arduino platform in this course}\cite{holstebro_education}.  
```

Comment:
To the footnote, maybe add that the first school we contacted for questions also used arduino. Also good source.